pet-store-vanilla-js/app.js

```diff
@@ -0,0 +1,125 @@
+import {
```

Comment:
I want loading / error indicators for all async requests.

Reply:
refreshing the pets table didn't have any indicator and i added it.
What do you mean by error indicators? Modal with the error?